src/Common/src/System/Drawing/ColorTable.cs

```diff
@@ -21,10 +22,15 @@ internal static class ColorTable
 
         private static void FillWithProperties(Dictionary<string, Color> dictionary, Type typeWithColors)
         {
+            object? value;
             foreach (PropertyInfo prop in typeWithColors.GetProperties(BindingFlags.Public | BindingFlags.Static))
             {
                 if (prop.PropertyType == typeof(Color))
-                    dictionary[prop.Name] = (Color)prop.GetValue(null, null);
+                {
+                    value = prop.GetValue(null, null);
```

Comment:
Color is an enum and prop.GetValue returns object?

FillWithProperties is only called in GetColors() on line 14, which doesn't seem to return null, that is why I added the Debug.Assert call

Reply:
A Debug.Assert is fine here, but it'd also be ok to just use `!`.  Personally I'd have done the latter, as it keeps the code cleaner and if the value could end up being null, the cast will fail and throw an exception highlighting the problem where and when it occurred, so the assert isn't adding a ton of benefit.